oss-cloud-frontend/src/components/ContributionsPage.vue

```diff
@@ -32,13 +50,19 @@ export default {
         {
           text: "Username",
           align: "left",
-          value: "owner"
+          value: "author",
         },
-        { text: "Repository", value: "repo" },
-        { text: "Number", value: "number" },
+        // {

```

Comment:
Do we need these comments?

Reply:
I was not sure about all the data that should be shown on the contributions page, so I left them commented until it is discussed.